FabLabKasse/faucardPayment/FAUcardPaymentThread.py

```diff
@@ -78,7 +79,8 @@ def __init__(self, dialog, amount, thread=QtCore.QThread.currentThread()):
         self.card_number = 0
         self.old_balance = 0
         self.new_balance = 0
-        self.amount = int(amount * 100)
+        self.amount = amount
```

Comment:
Changing the meaning of `amount` from cents to euros is dangerous. Maybe just remove the old `amount` variable and only use amount_cents?